### ISSUE TITLE
[Link and button labels] Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s. (08.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34876]
### Status
**[2024-07-12] [Fran]** This ticket on hold until the DS team componentizes the footer, at which time we can retest and close is issues have been resolved. Note that we (the Sitewide Team) should not close this ticket until issues are resolved.

#### VFS team name
Claims & Appeals

#### VFS product name
HLR V2

#### Point of Contact/Reviewers
Sarah Koomson (@Saraharaba8) - Accessibility
Josh Kim (@joshkimux) - Accessibility

---

### Platform Issue
Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s.

#### Links should not open in a new tab or window (WCAG G200)
[Reference our a11y presentation on links for more on user scenarios (google slides)](https://docs.google.com/presentation/d/14AwUoFOg_Ix1Txl9oePtPD1ntOpylBWwBJMYq9f5vmo/edit?usp=sharing)

##### Two scenarios may exist in which they _may_ be necessary. Specifically:

1. Opening a page containing context-sensitive information, such as help instructions, or an alternate means of completing a form, such as a calendar-based date picker, will significantly disrupt a multi-step workflow, such as filling in and submitting a form, if the page is opened in the same window or tab.
2. The user is logged into a secured area of a site, and following a link to a page outside of the secured area would terminate the user's logon. In this case opening external links in an external window allows the user to access such references while keeping their login active in the original window.

##### If necessary based on the prior scenarios

1. [Provide advanced warning that this will open in a new tab using visible link text (G201)](https://www.w3.org/TR/WCAG20-TECHS/G201.html). For example, `Your VA Welcome Kit (opens in new window)`
2. If there is not enough space to accommodate the longer `(opens in new window)` string, consider using an icon with an `aria-label` so sighted users still have an indication that this may open in a new window.
3. If there is no space at all to accommodate an icon or visible text, use an `aria-label` 

### Link, screenshot or steps to recreate
![D57E754C-A26C-46C8-A7FA-6F283EA2EF6A_1_105_c](https://user-images.githubusercontent.com/95157695/149583327-1b25c4ce-ede3-419d-8e64-0be894ce51c3.jpeg)

### VA.gov Experience Standard
[Category Number 08, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.6_AA

---

### Platform Recommendation
Include screen reader only elements to links

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by Mottie** [comment]
I discussed this issue with @Saraharaba8, and it appears that this ticket is related to the page footer links. We should inform the site-wide team if I this is correct.

Our HLR form does include some download links within the additional info 
expandable components opened by "Why isn’t my issue eligible?" and "Don’t see the issue you’re looking for?" links, so this may still apply to those since no `aria-label` was included, but I did follow this previous recommendation (https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340).

**Comment by shiragoodman** [comment]
@brianalloyd per the comments above and conversation with Sitewide Content, I've added your teams label to this ticket. As always, please do not close this out until the issue has been resolved. If you have questions, please let me know. Thanks!

**Comment by brianalloyd** [comment]
Thanks @shiragoodman this is definitely in our wheelhouse and will prioritize ticket accordingly. Thanks for looping me into the details.

**Comment by joshkimux** [comment]
@Mottie I've removed `hlr` label from here as I've double checked the download links and they're looking golden. Please continue following Jen's awesome guidance 👏 

<img width="840" alt="Screen Shot 2022-05-12 at 10 50 51 AM" src="https://user-images.githubusercontent.com/14154792/168106365-78d22bc5-bf33-40f7-b281-db57d18c6967.png">

@Saraharaba8 thanks for flagging this! This is a great catch 💯 

I've made small tweaks to the recommendation based on previous fixes we've made on VA.gov and [WCAG best practice guidance](https://www.w3.org/TR/WCAG20-TECHS/G200.html). Specifically:

## Links should not open in a new tab or window (WCAG G200)
[Reference our a11y presentation on links for more on user scenarios (google slides)](https://docs.google.com/presentation/d/14AwUoFOg_Ix1Txl9oePtPD1ntOpylBWwBJMYq9f5vmo/edit?usp=sharing)

### Two scenarios may exist in which they _may_ be necessary. Specifically:

1. Opening a page containing context-sensitive information, such as help instructions, or an alternate means of completing a form, such as a calendar-based date picker, will significantly disrupt a multi-step workflow, such as filling in and submitting a form, if the page is opened in the same window or tab.
2. The user is logged into a secured area of a site, and following a link to a page outside of the secured area would terminate the user's logon. In this case opening external links in an external window allows the user to access such references while keeping their login active in the original window.

### If necessary based on the prior scenarios

1. [Provide advanced warning that this will open in a new tab using visible link text (G201)](https://www.w3.org/TR/WCAG20-TECHS/G201.html). For example, `Your VA Welcome Kit (opens in new window)`
2. If there is not enough space to accommodate the longer `(opens in new window)` string, consider using an icon with an `aria-label` so sighted users still have an indication that this may open in a new window.
3. If there is no space at all to accommodate an icon or visible text, use an `aria-label`

**Comment by shiragoodman** [comment]
hey @joshkimux  - again, if this isn't related to HLR, we need to know what product is is related to. please let me know!

**Comment by joshkimux** [comment]
Attached sitewide here too!

**Comment by laflannery** [comment]
@FranECross I've reviewed this issue and it's referring to links in the footer - if they open in a new how, how they should be showed and coded.

Based on the current state of things and the fact that this is soon-ish (relatively speaking) going to be a component built by the DST, I want to say that:
1. We aren't going to be making any changes to the current footer right now **and**
2. We will not be managing the state of the footer in the future because that will be owned by DST.

Assuming those 2 things are accurate - what should we do with this ticket? Move it to blocked? Are we able to close it? It is already 2 years old at this point

**Comment by FranECross** [comment]
@laflannery Thanks so much for reviewing. I'll Ice Box the ticket until the footer is enhanced/componentized/requests in this ticket are moot and then we can close the ticket.

**Comment by laflannery** [comment]
@FranECross Should this be moved to DST folks?

**Comment by FranECross** [comment]
@laflannery Good call. I'll post the link to Carol and ask that they transfer it. Thanks!

**Comment by FranECross** [comment]
Messaged Carol in Slack asking her to take this ticket https://dsva.slack.com/archives/C01DBGX4P45/p1730221830069189

**Comment by randimays** [comment]
@FranECross I didn't find any links in the footer that open in a new tab; not sure this ticket still needs to be open.

**Comment by FranECross** [comment]
Closing as a no op; per Randi's review, there aren't currently any links in the footer that open in a new tab.
